apps/fhir/bluebutton/views/home.py

```diff
@@ -68,7 +68,7 @@ def fhir_conformance(request, via_oauth=False, *args, **kwargs):
     # Append Security to ConformanceStatement
     security_endpoint = build_oauth_resource(request, format_type="json")
     od['rest'][0]['security'] = security_endpoint
-    od['format'] = ['appliction/json']
+    od['format'] = ['application/fhir+json']
```

Comment:
Could you also add `'application/json'` to the array since we do accept it as a content type?